test/fuzz/luaL_loadbuffer/preamble.lua

```diff
@@ -22,9 +22,27 @@ local __index = function(self, key)
     end
     return always_number(key)
 end
+local __le = function(v1, v2)
+    if type(v1) == 'number' and type(v2) == 'number' then
+        return v1 <= v2 -- Numeric comparison.
+    elseif type(v1) == 'string' and type(v2) == 'string' then
+        return v1 <= v2 -- Lexicographic comparison.
+    else
+        return always_number(v1) <= always_number(v2)
+    end
+end
 local __len = function(_v)
     return DEFAULT_NUMBER
 end
+local __lt = function(v1, v2)
+    if type(v1) == 'number' and type(v2) == 'number' then
+        return v1 < v2 -- Numeric comparison.
+    elseif type(v1) == 'string' and type(v2) == 'string' then
+        return v1 < v2 -- Lexicographic comparison.
+    else
+        return always_number(v1) <= always_number(v2)
```

Comment:
Nit: I suppose this should be `<` for consistency.

Reply:
Fixed.

Reply:
```diff
--- a/test/fuzz/luaL_loadbuffer/preamble.lua
+++ b/test/fuzz/luaL_loadbuffer/preamble.lua
@@ -44,7 +44,7 @@ local __lt = function(v1, v2)
     elseif type(v1) == 'string' and type(v2) == 'string' then
         return v1 < v2 -- Lexicographic comparison.
     else
-        return always_number(v1) <= always_number(v2)
+        return always_number(v1) < always_number(v2)
     end
 end
 local __mod = function(v1, v2)
```